src/app/shared/components/navbar/navbar.component.scss

```diff
@@ -5,12 +5,22 @@
 .navbar {
   position: relative;
   height: 100%;
-  font-size: 14px;
+  font-size: 16px;
 
   .navbar__spacer {
     flex: 1 1 auto;
   }
 
+  .navbar__menu__text {
```

Comment:
`.navbar__menu` does not exist but is used within the template. Either delete `.navbar__menu` in the template and rename this one to `.navbar__menu-text` or create `.navbar__menu`.